tests/providers/test_internet.py

```diff
@@ -87,3 +87,20 @@ def test_free_email_domain(self):
     def test_tld(self):
         tld = self.factory.tld()
         assert tld in self.provider.tlds
+
+
+class TestNlNl(unittest.TestCase):
+
+    def setUp(self):
+        self.factory = Factory.create('nl_NL')
+        self.provider = self.factory.provider('faker.providers.internet')
+
+    def test_uuid4_email(self):
+        email = self.factory.uuid4_email()
+        validate_email(email, check_deliverability=False)
+
+    def test_uuid4_free_email(self):
```

Comment:
could you write tests for the `ascii_*` methods too?

Reply:
@fcurella: Sure.

Reply:
@fcurella:

ascii tests added.